account_receipt_journal/data/account_journal_data.xml

```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="utf-8" ?>
-<odoo noupdate="1">
-    <record id="sale_receipts_journal" model="account.journal">
```

Comment:
What happens if you have this module installed and there are account moves using these journals? Please preserve these XML IDs, at least for the main company.